GA/multi_point_problem_definition.py

```
@@ -254,25 +273,27 @@ def ComputeOmega(self,
         """
 
         if idx >= len(self.blade_blading_parameters[0]["RPS_lst"]):
-            raise IndexError(f"Expected at least {idx+1} RPS values, but got {len(self.blade_blading_parameters[0]['RPS_lst'])}")
+            raise IndexError(f"Expected at least {idx+1} RPS values, but got "
+                             f"{len(self.blade_blading_parameters[0]['RPS_lst'])}")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)